htdocs/install/page_dbsettings.php

```diff
@@ -129,13 +130,13 @@ function getDbCharsets($link) {
 function getDbCollations($link, $charset) {
 	static $collations = array( );
 
-	if ($result = exec_query("SHOW COLLATION WHERE Charset=" . quote_sql($charset), $link)) {
+	if ($result = exec_query("SHOW COLLATION WHERE Charset=" . quote_sql($charset, $link), $link)) {
 		while ($row = fetch_assoc($result)) {
```

Comment:
Should $link be here twice? quote_sql only takes one argument

Reply:
that looks like a merge mistake, I'll verify to be certain.